### ISSUE TITLE
Dockerize the auth0-simulator

### ISSUE THREAD

**Issue by dagda1** [thefrontside/simulacrum#180]
Having to install the auth0-simulator as a dev dependency is something we should try and avoid.

Creating a downloadable docker image that we could run 

```bash
docker run -i --rm --security-opt seccomp=$HOME/chrome.json \
    -v myvol:/app \
    --name auth0-simulator \
    frontside/auth-simulator:latest
```

We could try something like [this](https://dev.to/istarkov/fast-and-easy-way-to-setup-web-developer-certificates-450e) to make ssl configuration easier.

**Comment by cowboyd** [comment]
What does the user story look like for getting setup with SSL and a complete simulator?

**Comment by dagda1** [comment]
@cowboyd I linked to something I would like to spike in the above issue

https://dev.to/istarkov/fast-and-easy-way-to-setup-web-developer-certificates-450e

TLDR;

 - Register a DNS A record somewhere, something like `A auth0-simulator.com 127.0.0.1`
 - Use [certbot](https://certbot.eff.org/pages/about) to download the certs 
 - Point to these certs in `[https.ts](packages/server/src/https.ts)` rather than the `mkcert` certs.

```ts
      let ssl: SSLOptions = {
        key: fs.readFileSync(
          paths.ssl.keyFile
        ),
        cert: fs.readFileSync(paths.ssl.pemFile),
      } as const;
```
 
 I think we need some way of being able to configure the certs and by either defaulting to mkcert or letsencrypt certs.
